source/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -37,23 +37,40 @@ envoy_cc_library(
     visibility = ["//visibility:public"],
 )
 
+QUIC_HEADER_ONLY_IMPL = [
+    "quic_aligned_impl.h",
+    "quic_arraysize_impl.h",
+    "quic_containers_impl.h",
+    "quic_endian_impl.h",
+    "quic_estimate_memory_usage_impl.h",
+    "quic_export_impl.h",
+    "quic_fallthrough_impl.h",
+    "quic_flag_utils_impl.h",
+    "quic_iovec_impl.h",
+    "quic_prefetch_impl.h",
+    "quic_ptr_util_impl.h",
+    "quic_string_impl.h",
+    "quic_string_piece_impl.h",
+    "quic_uint128_impl.h",
+]
+
+genrule(
```

Comment:
Add brief comment explaining the reason for this. E.g:

```
# Generate dummy .cc file that includes all platform headers, to ensure that they
# will actually be compiled.
#
# TODO(wub): remove once real dependencies from core QUICHE code to platform
#   libraries are added.
```

Reply:
Done.